docs/tutorials/gradients.ipynb

```diff
@@ -604,9 +604,20 @@
       "source": [
         "## 4. Advanced usage\n",
         "Here you will learn how to define your own custom differentiation routines for quantum circuits.\n",
-        "All differentiators that exist inside of TensorFlow Quantum subclass `tfq.differentiators.Differentiator`. A differentiator must implement `differentiate_analytic` and `differentiate_sampled`.\n",
+        "All differentiators that exist inside of TensorFlow Quantum subclass `tfq.differentiators.Differentiator`. A differentiator must implement `get_gradient_circuits`, which tells the base class which circuits to measure to obtain an estimate of the gradient.\n",
```

Comment:
Might want to note here that you can also (optionally) overload `differentiate_analytic` or `differentiate_sampled`.

Reply:
Done